sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/UpdateComponentRequestOptions.java

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.digitaltwins.core;
+
+
+import java.util.List;
+
+/**
+ * Optional settings that are specific to calls to {@link DigitalTwinsClient#updateComponent(String, String, List)} and its overloads.
+ */
+public class UpdateComponentRequestOptions extends RequestOptions {
```

Comment:
Would you mind adding a similar class for the update relationship request options as well, and changing the API signature to take in the new inherited type?

Reply:
Sure thing